### ISSUE TITLE
standalone: pressing esc quits

### ISSUE THREAD

**Issue by increpare** [TerryCavanagh/boscaceoil#37]
without even a prompt. this is bad if you've forgotten to save a file.

**Comment by TerryCavanagh** [comment]
To do: Add "Are you sure you want to quit?" dialogue when user presses escape.

**Comment by ingdanko** [comment]
Can I bump this one? Would make a great and needed feature imo.

**Comment by TerryCavanagh** [comment]
Yeah, I'd really like to implement this one! Unfortunately it's probably not going to happen very soon (an explanation about why is here: https://github.com/TerryCavanagh/boscaceoil/issues/63)

**Comment by Eldakan** [comment]
I just had a song that I worked on 1,5hours and didnt save because I was stupid and in the flow get deleted because of this issue, even though I actually knew about it... Sucks

As your game will be released tomorrow you could maybe solve that issue now?
I love BC btw
